geoh5py/objects/curve.py

```diff
@@ -173,7 +173,7 @@ def validate_cells(self, indices: list | tuple | np.ndarray | None) -> np.ndarra
         if not np.issubdtype(indices.dtype, np.integer):
             raise TypeError("Indices array must be of integer type")
 
-        if np.max(indices) > self.n_vertices - 1:
+        if len(indices) > 1 and np.max(indices) > (self.n_vertices - 1):
```

Comment:
I don't get it, len(indices) will always be >1 no?
Don't you mean <=1?

Reply:
It's possible that all vertices get removed, which crashes the np.max()